content/desktop/extensions-sdk/dev/api/reference/interfaces/ExecStreamOptions.md

```diff
@@ -33,7 +33,7 @@ to true. The callback is then invoked once for each line.
 
 | Name | Type | Description |
 | :------ | :------ | :------ |
-| `data` | { `stdout`: `string` ; `stderr?`: `undefined`  } \| { `stdout?`: `undefined` ; `stderr`: `string`  } | output content. Can include either stdout string, or stderr string, one at a time. |
+| `data` | { `stdout`: `string` ; `stderr?`: `undefined`  } \| { `stdout?`: `undefined` ; `stderr`: `string`  } | Output content. Can include either stdout string, or stderr string, one at a time. |
```

Comment:
suggestion: I'd wrap these type signatures in a code element and drop the space before the semicolon delimiter

```suggestion
| `data` | `{ stdout: string; stderr?: undefined } \| { stdout?: undefined; stderr: string }` | Output content. Can include either stdout string, or stderr string, one at a time. |
```